src/external_api.py

```diff
@@ -0,0 +1,29 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+def convert_currency(transaction):
+    """Конвертирует сумму транзакции в рубли."""
+    amount = transaction.get('amount', 0)
+    currency = transaction.get('currency')
```

Comment:
Получать ключи нужно так (оба):
```suggestion
    currency = transaction['currency']
```
Чтобы в случае отсутствия ключа упала ошибка `KeyError`